paddle/pir/include/core/op_info.h

```diff
@@ -83,6 +84,8 @@ class IR_API OpInfo {
     return OpInfo(static_cast<OpInfoImpl *>(pointer));
   }
 
+  const std::vector<std::string> GetAttributesName() const;
```

Comment:
```suggestion
std::vector<std::string> GetAttributesName() const;
```
这里返回值用const修饰是否有意义呢？

Reply:
喔喔，没意义，忘记删了 :joy:，我下一个 PR 删掉吧～

Reply:
已经和其它问题一起删掉了～